### ISSUE TITLE
The lines do not stay in parallel (mil std C) when editing a water crossing bridge or gap 

### ISSUE THREAD

**Issue by alberto-acevedo** [missioncommand/emp3-web#351]
This shape as well as the block task are type super auto shape. In the case of the gap, the editor allows a free will of movement of the control points with no regards to keeping the lines in parallel.

**Comment by alberto-acevedo** [comment]
Waiting for pull request review.

**Comment by alberto-acevedo** [comment]
Merged into the dev branch. closing this issue.
